packages/components/select/src/use-select.ts

```diff
@@ -78,6 +78,9 @@ interface Props<T> extends Omit<HTMLNextUIProps<"select">, keyof SelectVariantPr
   startContent?: React.ReactNode;
   /**
    * Element to be rendered in the right side of the select.
+   * if you pass this prop and the `onClear` prop, the passed element
+   * will have the clear button props and it will be rendered instead of the
+   * default clear button.
```

Comment:
i think this statement is no longer correct. now the endContent is placed next to clear button. these 3 lines can be deleted.